src/controllers/tripsController.js

```diff
@@ -373,4 +359,67 @@ export default class requestController {
       return Response.errorResponse(res, 500, res.__('server error'));
     }
   }
+
+
+  /**
+   * @param  {object} req
+   * @param  {object} res
+   * @param  {object} next
+   * @return {object} view specific trip request
+   */
+  static async viewRequest(req, res) {
+    try {
+      const { user } = req;
+      const { requestId } = req.params;
+      if (user.role === 'manager') {
+        const availableRequest = await db.Request.findOne({
+          where: {
+            id: requestId,
+            managerId: user.id
+          }
+        });
+        if (!availableRequest) {
+          return Response.errorResponse(res, 404, res.__('Request not found or not yours to manage'));
+        }
+        return Response.success(res, 200, res.__('Request found'), availableRequest);
```

Comment:
hey, @Baraka-Mugisha did you test for when a manager wants to view an individual request?

Reply:
I added the test